src/app/settings/initial-settings.ts

```diff
@@ -19,6 +19,6 @@ export const initialSettingsState: Settings = {
   organizerColumnsGhost: ['icon', 'name', 'locked', 'tag', 'perks', 'notes'],
   compareBaseStats: false,
   sidecarCollapsed: false,
-  activeMode: false,
+  activeMode: true,
```

Comment:
FYI I swapped it to be on by default -- that way when someone flips to the single character mode for the first time, they'll see the sidebar and can close it, but at least will know it's there. I'm fine w/ reverting it, but wanted to call it out

Reply:
I think I'd prefer if this is closed by default